int32 target_angle
---
bool success
---
int32 current_angle